gslab_scons/tests/_side_effects.py

```diff
@@ -34,8 +34,8 @@ def side_effect(*args, **kwargs):
             source = match.group('source')
             log    = '%s.Rout' % re.sub('\.R', '', source)
     
-        if executable == "R CMD BATCH" and log:
-            with open(log.strip(), 'wb') as log_file:
+        if executable == "Rscript" and log:
```

Comment:
But I think I do need to change the above definition of log when log is missing.